hbase-server/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupInfoManagerImpl.java

```diff
@@ -1015,17 +1016,17 @@ private void moveServerRegionsFromGroup(Set<Address> movedServers, Set<Address>
               assignmentFutures.add(Pair.newPair(region, future));
             } catch (IOException ioe) {
               failedRegions.add(region.getRegionNameAsString());
-              LOG.debug("Move region {} from group failed, will retry, current retry time is {}",
-                region.getShortNameToLog(), retry, ioe);
+              LOG.debug("Move region {} from server {} failed, will retry, current retry time is {}",
+                region.getShortNameToLog(), owner.getHostname(), retry, ioe);
               toThrow = ioe;
             }
           }
         }
       }
-      waitForRegionMovement(assignmentFutures, failedRegions, targetGrp.getName(), retry);
+      waitForRegionMovement(assignmentFutures, failedRegions, sourceGroupName, retry);
       if (failedRegions.isEmpty()) {
-        LOG.info("All regions from server(s) {} moved to target group {}.", movedServerNames,
-          targetGrp.getName());
+        LOG.info("All regions from server(s) {} moved to source group {}.", movedServerNames,
```

Comment:
Why not say, 'All regions from {} moved from {} to {}, movedServerNames, sourceGroupName, targetGroupName?

Reply:
The regions in movedServerNames originally belongs to the source group. So it should not say "move from sourceGroupName to targetGroupName". Maybe it can be like this: “All regions from {} are moved back to {}, movedServerNames, sourceGroupName”?

Reply:
If that is what is happening, then yes, your rephrasing makes more sense (IMO). Thanks.